### PR TITLE
Updated swagger json

### PR DESCRIPTION
**Pull request by tobiasgpsgate**
Updated json using steps described in [REAME.md](https://github.com/gpsgate/apidocs/blob/master/README.md)

See related PR for new Update Device endpoint: https://github.com/gpsgate/gpsgate/pull/12073

The json contains more updates than only the new Update Device endpoint

### CODE REVIEW

**Comment by tobiasgpsgate** [comment]
I followed the instructions in README.md. I couldn't find plugin Passenger Manager. Is that still a thing?

The swagger documentation I generated contains things that isn't on the existing json, such as Site, Locale, SMS
and probably more as well.

How should I continue? What's the process of updating this document? What is expected to exist in the end result ?

@GreggGut @bjornhandersson

**Comment by GreggGut** [comment]
> I followed the instructions in README.md. I couldn't find plugin Passenger Manager. Is that still a thing?
> 
> The swagger documentation I generated contains things that isn't on the existing json, such as Site, Locale, SMS and probably more as well.
> 
> How should I continue? What's the process of updating this document? What is expected to exist in the end result ?
> 
> @GreggGut @bjornhandersson

The docs have not been updated in a little while hence the missing part. Great that is has been added now.

Passenger Manager has been deprecated and recently removed. It should not be in the API docs.

**Comment by tobiasgpsgate** [comment]
> > I followed the instructions in README.md. I couldn't find plugin Passenger Manager. Is that still a thing?
> > The swagger documentation I generated contains things that isn't on the existing json, such as Site, Locale, SMS and probably more as well.
> > How should I continue? What's the process of updating this document? What is expected to exist in the end result ?
> > @GreggGut @bjornhandersson
> 
> The docs have not been updated in a little while hence the missing part. Great that is has been added now.
> 
> Passenger Manager has been deprecated and recently removed. It should not be in the API docs.

Ok, then it sounds like this generated json is what we expect, I will promote this draft to a real PR then

**Comment by GreggGut** [comment]
Looking at the code I see that some of those are private APIs. 
@bjornhandersson @janagpsgate @AlexP-SSA

File: REST/Documentation.json [hunk 1]
```diff
@@ -6011,14 +6103,14 @@
       },
       "parameters": []
     },
-    "/applications/{applicationid:int}/users/{userid:int}/fuelconsumption": {
+    "/applications/{applicationid:int}/accumulators/{accumulatorid:long}": {
       "get": {
         "tags": [
-          "FuelConsumption"
```
Comment (janagpsgate) [inline REST/Documentation.json:6017]: Fuel consumption seems to be gone?

Comment (tobiasgpsgate) [reply REST/Documentation.json:6017]: Hmm, I'm fairly sure I saw it in my local swagger, I'll take a look

**Comment by GreggGut** [comment]
@tobiasgpsgate In order to get the API docs the users get to see you should install your changes on a VM and get the docs from there.
I completely forgot about the private APIs. The reason some of those new APIs showed up is that if you run the service in debug mode where we expose more internal APIs. You have an example [here](https://github.com/gpsgate/gpsgate/blob/master/GpsGateOnline/dotNET/API/GpsGate.API.Resources/Settings/Locale/LocaleMetadataModule.cs#L2)

Having some automation around this would be idea, any suggestions?

File: REST/Documentation.json (hunk 1 above)
Comment (tobiasgpsgate) [reply REST/Documentation.json]: I did the process again and uploaded an updated version, it's there now

**Comment by tobiasgpsgate** [comment]
> @tobiasgpsgate In order to get the API docs the users get to see you should install your changes on a VM and get the docs from there. I completely forgot about the private APIs. The reason some of those new APIs showed up is that if you run the service in debug mode where we expose more internal APIs. You have an example [here](https://github.com/gpsgate/gpsgate/blob/master/GpsGateOnline/dotNET/API/GpsGate.API.Resources/Settings/Locale/LocaleMetadataModule.cs#L2)
> 
> Having some automation around this would be idea, any suggestions?

Ok, yes I did this from my local machine.

For automation, can't we fetch this from a live cloud instance ?

**Comment by AlexP-SSA** [comment]
> Looking at the code I see that some of those are private APIs

yep
at least these api's are private for sure:
/settings/locale
/cloudsetup/sms
/cloudsetup/passwordreset

---
>I completely forgot about the private APIs. The reason some of those new APIs showed up is that if you run the service in debug mode where we expose more internal APIs. You have an example [here](https://github.com/gpsgate/gpsgate/blob/master/GpsGateOnline/dotNET/API/GpsGate.API.Resources/Settings/Locale/LocaleMetadataModule.cs#L2)
>
> Having some automation around this would be idea, any suggestions?

as far as I remember there are 2 "types" of private api's:
- those which are not shown on a swagger page at all (and I believe they are not a problem in this case)
- those which are show in a debug mode only

so I think maybe we could run the project in a release mode from visual studio to get a correct file?

**Comment by tobiasgpsgate** [comment]
Why do we need this copy on the main web site? Why can't we just point people to a real server where the swagger is generated already?

**Comment by tobiasgpsgate** [comment]
Updated json, now generated from VM, It definitely looks cleaner now.

Passengers endpoints and the following endpoint seems to have been removed
POST /applications/{applicationid:int}/tokens 

Is this reasonable?

**Comment by GreggGut** [comment]
> Why do we need this copy on the main web site? Why can't we just point people to a real server where the swagger is generated already?

We have a developer section on our website and it's there so that we can showcase potential customers what we have as a pubic API.

**Review by GreggGut — APPROVED** [review]

**Pull request merged**